requirements-dev.txt

```diff
@@ -5,7 +5,7 @@ flake8==3.8.2
 nose==1.3.7
 splinter>=0.13.0
 -e git+https://github.com/qld-gov-au/ckanext-datarequests@training#egg=ckanext-datarequests
--e git+https://github.com/qld-gov-au/ckanext-data-qld@develop#egg=ckanext-data_qld
+-e git+https://github.com/salsadigitalauorg/ckanext-data-qld@develop#egg=ckanext-data_qld
```

Comment:
have we merged the latest ckenext-data-qld into our version yet?

Reply:
tho please change this to qld-gov-au (or we may need to remember to do that post merge)